File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    // ask user for input on how high (s)he wants the pyramid to be
    do
    {
        height = get_int("height is:");
    }
    while (height < 1 || height > 8);

    // for the entire height print spaces first and then hashes so that the pyramid looks reversed
    for (int i = 0; i < height; i++)
    {
        for (int space = 0; space > i - height + 1; space--)
        {
            printf(" ");
        }
        for (int hash = 0; hash < i + 1; hash++)
        {
            printf("#");
        }
        printf("  ");

        // now print the same but without spaces so that the pyramid mirrors itself
        for (int hash = 0; hash < i + 1; hash++)
        {
            printf("#");
        }
        printf("\n");

    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    size = get_int("what is the desired size of the pyramid?:")
    if (size > 0 and size < 9):
        break
space = 0
hash = 0
for i in range(size):
    for space in range(0, i - size + 1, -1):
        print(" ", end="")
    for hash in range(0, i + 1):
        print("#", end="")
        hash += 1
    print("  ", end="")
    for hash in range(0, i + 1):
        print("#", end="")
        hash += 1
    print("")

